docs/standards.md

```diff
@@ -1,42 +1,38 @@
 # Standards
 
-### Language Style Guides
+## Language Style Guides
 
-### For All source files
+### For all source files
 Each source code file needs to contain the license which it is licensed under.
 
-#### For C
+### For C / C++ / C#:
```

Comment:
I remember @andrew-hirata-playeveryware mentioned this section was intended for C (and probably C++)?

Reply:
Yeah - this is true - but of the standards that are actually listed on this page the same standards tend to apply, and previously there were empty sections for C and C++, with the only standards written beneath the "C#" section.